suite/runsuite_wrapper.pl

```diff
@@ -249,6 +249,7 @@
                 'code_api|tool.drcacheoff.windows-invar' => 1, # i#6599
                 'code_api|tool.drcacheoff.invariant_checker' => 1, # i#6599
                 'code_api|tool.drcacheoff.getretaddr_record_replace_retaddr' => 1, # i#6599
+                'code_api|tool.record_filter' => 1, # i#6599
```

Comment:
There's no record of this test ever failing that I can easily see: I don't see it in #6599, nor are there recent xref links from other PR's or issues.  Please update #6599 pointing to where you are seeing this test fail.

Reply:
Adding that info the PR description would also help: is this only the post-merge, e.g., is key info.

Reply:
> Adding that info the PR description would also help: is this only the post-merge, e.g., is key info.

Yes I found logs (see my other comment) in the post-merge long suite.

Reply:
Since tool.record_filter is post-merge only, no PR would encounter the failure and have a reason to xref i#6599. This PR that adds it to the ignore list would be the first one to do so. Adding more details to the PR description (as suggested by both review comments) would help establish that link for future readers.

Reply:
nit: I would expect the issue to be updated before sending this PR.

Reply:
The issue is automagically updated already. I wasn't expecting to repeat the info.
[Certainly updating the PR description is a good idea.]

Reply:
PR description updated

Reply:
The main request here was to update the issue #6599.

Reply:
Done